src/blenderbim/blenderbim/core/georeference.py

```diff
@@ -0,0 +1,99 @@
+# BlenderBIM Add-on - OpenBIM Blender Add-on
+# Copyright (C) 2022 Dion Moult <dion@thinkmoult.com>
+#
+# This file is part of BlenderBIM Add-on.
+#
+# BlenderBIM Add-on is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# BlenderBIM Add-on is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with BlenderBIM Add-on.  If not, see <http://www.gnu.org/licenses/>.
+
+import bpy
+import blenderbim.bim.helper
+
+from blenderbim.bim.module.georeference.data import GeoreferenceData
+
+from math import radians, degrees, atan, tan, cos, sin
```

Comment:
Core should have no imports. All these details should be in a tool instead. See how other cores work :)

Reply:
Ok, i'll try and i'll tell you if i can't make it.

Reply:
Ok, i'll fix.